cumulusci/files/templates/project/scratch_def.json

```diff
@@ -1,10 +1,11 @@
 {
   "orgName": "{{ package_name }} - {{ org_name }}",
   "edition": "{{ edition }}",
-  "orgPreferences" : {
-    "enabled": [
-      "S1DesktopEnabled",
-      "ChatterEnabled"
-    ]
+  "settings": {
+    "orgPreferenceSettings": {
+      "s1DesktopEnabled": true,
+      "chatterEnabled": true,
+      "translation": true
```

Comment:
See my other comment -- this means that we want translation on for some scratch orgs but not others. Which means we either need two different template files, or a jinja2 conditional within the template based on a variable that we pass when rendering it.

Reply:
I'd say the variable should be to signify an unmanaged vs a managed environment.  The dev and feature org shapes are unmanaged, beta and release are managed.